src/api/posts/src/routes/posts.js

```diff
@@ -9,7 +9,7 @@ const posts = Router();
 
 const prepareData = async (ids, shouldExpand = false) => {
```

Comment:
Get rid of `async` here, we aren't going to `await` anymore.

Reply:
Got it. A new commit is pushed.